src/utils/__test__/ConfigUtils.spec.js

```diff
@@ -91,3 +91,119 @@ describe('getConversionMethod with possible values', () => {
     }
   );
 });
+
+describe('Check createNewTableLine returns value', () => {
+  const COLUMNS = [
+    // Basic webapp handler DONE
```

Comment:
Glad it's "DONE", but maybe we can remove it from the comment now? :smile:

Reply:
DONE